### PR TITLE
Make sure that serivce account -1 is created with full permission for test and dev

### PR DESCRIPTION
**Pull request by Torbacka**
I'm not sure where the test docker-compose is used but I added TEST_SERVICE_TOKEN there as well.

### CODE REVIEW

File: api/src/firstrun.py [hunk 1]
```diff
@@ -64,8 +64,8 @@ def create_admin(admins):
     banner(BLUE, "Admin User")
 
     s = input("Do you want to create a new admin user"
-              " (you can later use the create_user.py script to create users)? [Y/n]: ")
-    if s not in {"", "y", "yes"}:
+              " (you can later use the create_user.py script to create users)? [y/n]: ")
+    if s not in {"", "y", "Y", "yes"}:
```
Comment (emanuelen5) [inline api/src/firstrun.py:68]: Big letter *Y* just means it is default (if left blank). But we can make it case insensitive just for the heck of it 🙂 

```suggestion
              " (you can later use the create_user.py script to create users)? [Y/n]: ")
    if s.lower() not in {"", "y", "yes"}:
```

**Comment by emanuelen5** [comment]
Adding @andersroos as reviewer. He understands the docker-compose better.

File: api/src/init_db.py [hunk 2]
```diff
@@ -38,6 +38,7 @@ def refresh_service_access_tokens(session_factory):
                         user_id=service_user.id,
                         access_token=service_user.token or generate_token(),
                         browser='',
+                        permissions=",".join(service_user.permissions),
```
Comment (andersroos) [inline api/src/init_db.py:41]: Why is this needed, what is it with the current solution that does not work?

File: docker-compose.dev.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@ services:
       - "8010:80"
     environment:
       DEV_RUN: "true"
-      TEST_SERVICE_TOKEN:
+      TEST_SERVICE_TOKEN: "${TEST_SERVICE_TOKEN}"
```
Comment (andersroos) [inline docker-compose.dev.yml:18]: Why is this needed, don't the line before the change do exactly the same thing?

Comment (andersroos) [inline docker-compose.dev.yml:18]: Why is this needed, don't the line before the change do exactly the same thing?

Comment (andersroos) [inline docker-compose.dev.yml:18]: Does this change do anything? Doesn't an empty env variable means that it will pick it from the env?

File: docker-compose.test.yml [hunk 4]
```diff
@@ -12,7 +12,7 @@ services:
     environment:
       TEST: "true"
       CORS_ALLOWED_ORIGINS: 'http://admin,http://api,http://public'
-      TEST_SERVICE_TOKEN:
+      TEST_SERVICE_TOKEN: "${TEST_SERVICE_TOKEN}"
```
Comment (andersroos) [inline docker-compose.test.yml:15]: Same for this?

**Comment by emanuelen5** [comment]
Stale
